docs/website/docs/building-from-source/getting-started.md

```diff
@@ -44,8 +50,28 @@ git submodule update --init
 
 Configure then build all targets using CMake:
 
+Configure CMake:
+
+``` shell
+# Recommended for simple development using clang and lld:
+cmake -GNinja -B ../iree-build/ -H . \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+    -DIREE_ENABLE_ASSERTIONS=ON \
+    -DCMAKE_C_COMPILER=clang \
+    -DCMAKE_CXX_COMPILER=clang++ \
+    -DCMAKE_LINKER=lld
+
+# Minimal/release with system compiler:
+cmake -GNinja -B ../iree-build/ -H . -DCMAKE_BUILD_TYPE=Release
```

Comment:
you shouldn't need to specify either build type or ninja here (and without ninja multibuild configured having a different type here is an error) as those are active only during configuration above

Reply:
Reworked.